packages/trader/src/Modules/CFD/Containers/cfd-change-password-confirmation.tsx

```diff
@@ -13,7 +13,12 @@ const ChangePasswordConfirmation = ({
     onConfirm,
     onCancel,
 }: TCFDChangePasswordConfirmationProps) => (
-    <Formik initialValues={{}} onSubmit={onConfirm}>
+    <Formik
+        initialValues={{
```

Comment:
initialValues is a required Formik prop so despite we don't input any values in this component, we still have to add initial value here in order to avoid a TS error: "Property 'password' is missing in type '{}' but required in type 'TCFDPasswordFormValues'". TCFDPasswordFormValues is a type of object { password: string } that is expected in the parent component. The value of this exact 'password' will not be used in the parent component, that's why this change doesn't have any affect on the behaviour.